### PR TITLE
Display deployment dialog from remote debug window

### LINKED ISSUE #307: Expose Deployment Dialog from Remote Debug window
When the remote debug command fails to find an app running in TAS with the same name as the selected .NET project, it prompts to either select an existing app of a different name or push a new version of the selected app. The Deployment options for the "push new" process are much more limited than those provided in the Deployment dialog. With a different design, the remote debug window could display the Deployment dialog in full & use the same mechanisms as usual for pushing an app

### PR DESCRIPTION
**Pull request by atwoosnam**
resolves #307

### CODE REVIEW

**Comment by atwoosnam** [comment]
This issue seems to exist on `main` already! (see #319) 
Re-opening this PR as it didn't introduce this issue

File: src/Services/src/Project/IProjectService.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿namespace Tanzu.Toolkit.Services.Project
+{
+    public interface IProjectService
+    {
+        string ProjectName { get; set; }
+        string PathToProjectDirectory { get; set; }
+        string TargetFrameworkMoniker { get; set; }
+    }
+}
```
Comment (ccheetham) [inline src/Services/src/Project/IProjectService.cs:9]: missing newline

**Review by ccheetham — CHANGES_REQUESTED** [review]
minor formatting changes

**Review by ccheetham — APPROVED** [review]
lgtm

**Pull request merged**